### DISCUSSION TITLE
cannot install pimcore (docker | imagemagick)

### DISCUSSION THREAD

**Discussion by patriciomg** [pimcore/pimcore#14040] [Q&A / Support] [answered]
Hi everyone, 
pimcore resists with all its power to get installed and configured. My Machine is a mac pro with silicon chip if that matters.
So far, using the docker/skeleton, files and dirs are deployed and when I try:

`docker-compose exec php vendor/bin/pimcore-install --mysql-host-socket=db --mysql-username=pimcore --mysql-password=pimcore --mysql-database=pimcore --ignore-existing-config
`

I get:

`php: symbol lookup error: /usr/local/lib/libMagickCore-7.Q16HDRI.so.10: undefined symbol: heif_init`

sadly, there is no help that google can show me. 
So I am stuck on this.. :-/ 

note:
I already did
`apt-get update && apt-get install -y imagemagick`

thanks in advance

**Comment by patriciomg** [comment]
current installed version of imagemagick is 8:6.9.11.60+dfsg-1.3

**Comment by jdreesen** [comment, marked as answer]
Please try `v1.2` of the pimcore docker image. It should have been fixed by pimcore/docker#120 (see: pimcore/docker#119)

**Reply by patriciomg** [reply]
Hi, 
thank your for your answer. That worked fine 👍 

now i just have to solve another problem with the connection to the Mysql Container 🤣 
"An exception occurred in driver: SQLSTATE[HY000] [2002] No such file or directory"
I will follow this but in another thread. 

Thank you again for your help

Pat.
